NEWS.md

```diff
@@ -4,7 +4,7 @@ Julia v1.12 Release Notes
 New language features
 ---------------------
 
-- New option `--trim` for building "trimmed" binaries, where code not provably reachable from entry points
+- New option `--trim` for building "trimmed" binaries, where code provably not reachable from entry points
   is removed. Entry points can be marked using `Base.Experimental.entrypoint` ([#55047]).
```

Comment:
How about a rewording like this?

```suggestion
- New option `--trim` creates smaller binaries by removing code that was not proven to be reachable
 from the entry points. Entry points can be marked using `Base.Experimental.entrypoint` ([#55047]).
```

Reply:
That wording seems fine but is still slightly inverted: it should be "proven not to be reachable" rather than "not proven to be reachable"

Reply:
Ah, I only now saw the other comments above. That's kinda scary that it is indeed inverted as described above.